manubot/cite/citeproc.py

```diff
@@ -51,20 +51,25 @@ def get_jsonschema_csl_validator():
     import jsonref
     import jsonschema
 
-    url = 'https://github.com/dhimmel/schema/raw/manubot/csl-data.json'
+    url = 'https://github.com/dhimmel/csl-schema/raw/empty-date-parts/csl-data.json'
```

Comment:
What's the difference between these two JSON files?  Does `empty-date-parts` have some short term customization that won't be part of the `manubot` schema?

Reply:
Before merging we will revert this PR is using the manubot CSL branch. However, I will add this https://github.com/dhimmel/csl-schema/commit/b0128abc7f7dd78bf140607f339273966fcc1bba from `empty-date-parts` onto that branch.

Basically, I needed to test this PR with the new CSL JSON Schema.